src/views/ApartmentApp/components/StaffMenu/index.js

```diff
@@ -117,7 +109,7 @@ const StaffMenu = ({ userProfile, authentication }) => {
                     variant="contained"
                     color="primary"
                     startIcon={<GetAppIcon />}
-                    disabled={!authentication}
+                    disabled={applicationJsonArray?.length < 1} // This check works correctly for both Number and null
```

Comment:
I think the comment doesn't need to stay here long-term. It's good to test it, but since that's a feature of JavaScript it doesn't necessarily require a comment.

Reply:
Agreed

Reply:
```suggestion
                    disabled={applicationJsonArray?.length < 1}
```